src/miner.cpp

```diff
@@ -370,7 +363,7 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const MinerAddre
         while (!vecPriority.empty())
         {
             // Take highest priority transaction off the priority queue:
-            double dPriority = vecPriority.front().get<0>();
+            // double dPriority = vecPriority.front().get<0>();
```

Comment:
Stray comment left in the code. Should the `TxPriority` data structure be modified to remove the priority element? Also, as a nit I think that `TxPriority` would be better as a struct with named elements rather than a tuple.

Reply:
i've removed priority from `TxPriority` but kept it as a tuple.